stdlib/system/src/main/ballerina/system/system.bal

```diff
@@ -40,3 +40,71 @@ public function getUserHome() returns string = external;
 #
 # + return - The random string
 public function uuid() returns string = external;
+
+# Reports whether file or directory exists for the given the
+# path.
+#
+# + path - String value of file path.
+# + return - True if path is absolute, else false
+public function exists(string path) returns boolean = external;
+
+# Creates a new directory with the specified file name.
+# If parentDirs flag is true, Creates a directory in specified
+# path with any necessary parents.
+#
+# + dir - directory name.
+# + parentDirs - Indicates whether the `createDir` should create non-existing parent directories.
+# + return - Returns absolute path value of the created directory or an `error` if failed
+public function createDir(string dir, boolean parentDirs = false) returns error? = external;
+
+# Removes the specified file or directory.
+# If recursive flag is true, Removes the path and any children  # it contains.
+#
+# + path - String value of file/directory path.
+# + recursive - Indicates whether the `remove` should
+# recursively remove all the file inside the given directory.
+# + return - Returns an `error` if failed to remove.
+public function remove(string path, boolean recursive = false) returns error? = external;
+
+# Renames(Moves) old path to new path.
+# If new path already exists and it is not a directory, this
+# replaces the file.
+#
+# + oldpath - String value of old file path.
+# + newpath - String value of new file path.
+# + return - Returns an `error` if failed to rename.
```

Comment:
Better to add an empty docline between the parameters and the return value.

Reply:
In stdlib, we didn't add a new line between parameters and return value. So I followed the same convension here